client/app/scripts/components/app.js

```diff
@@ -46,6 +46,9 @@ class App extends React.Component {
       getTopologies(this.props.activeTopologyOptions, this.props.dispatch);
     }
     getApiDetails(this.props.dispatch);
+    if (this.props.contrastMode) {
+      this.props.toggleContrastMode(true);
+    }
```

Comment:
This only fires once when the application load. It is mainly just there to ensure that if `{ contrastMode: true }` is in the URL it will get toggled.